example/eblob_backend.cpp

```diff
@@ -198,6 +198,79 @@ int blob_file_info_new(eblob_backend_config *c, void *state, dnet_cmd *cmd) {
 	return 0;
 }
 
+int blob_del_new(eblob_backend_config *c, dnet_cmd *cmd, void *data) {
```

Comment:
What does it print from "information how and why a record was removed: was it regular remove or with cas, was the cas applied or a record is corrupted or it doesn't have ext header etc. Also it should measure and write in this log a time spent on checksum verification."